src/components/FilterDropdown/FilterDropdown.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { useUserData } from "../../context/user-data-context";
+import styles from "./FilterDropdown.module.css";
+
+
+export const FilterDropdown = (props) => {
+    const {userData} = useUserData();
+    const {filterState, setFilterState} = props;
+    return (
+      <>
+        <div className={styles["filter-dropdown"]}>
+          <label>Filter by Tag : </label>
+          <select name="tags" value={filterState} onChange={(e) => setFilterState(e.target.value)}>
+            <option value="All">
+              All
+            </option>
+            {userData.allTags.map((item) => (
+              <option key={item} value={item}>
+                {item}
+              </option>
+            ))}
+          </select>
```

Comment:
Must change: tags are not appearing in the filter drop down menu. Re-check.

Reply:
Did you press enter after you typed a tag?
Since a task can have multiple tags, you need to press enter after you type a tag for it associated with a task and hence added to the list.

So essentially the tags that you typed did not get added to the task, hence there were no other tags to be displayed on the dropdown.
Thanks for letting me know about this, this is a UI concern, I need to add something to let people know that tags are only associated after pressing enter.